themes/default/content/about/leadership.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Leadership
+meta_desc: Meet Pulumi's leadership team. Pulumi is a Seattle-based company that enables teams to create, deploy, and manage modern cloud applications and infrastructure.
```

Comment:
this isn't the latest pulumi meta_desc. can you look at what's used in pulumi.com? apply this comment to other instances of this meta_desc

Reply:
Done in the latest revision.